web/apps.py

```
@@ -1,10 +1,34 @@
     default_auto_field = "django.db.models.BigAutoField"
     name = "web"
 
     def ready(self):
-        """Import signals when the app is ready"""
+        """Import signals when the app is ready."""
+        self._patch_django_context_copy_for_python_314_tests()
+
         import web.signals  # noqa
+
+    @staticmethod
+    def _patch_django_context_copy_for_python_314_tests():
```

Ruff (0.15.5)

[warning] Missing return type annotation for staticmethod `_patch_django_context_copy_for_python_314_tests`

Add return type annotation: `None`

(ANN205)